web/developer.actyx.com/docs/conceptual/performance-and-limits.mdx

```diff
@@ -140,24 +128,11 @@ Machine connectors that only make machine state and counters available in the Ac
 The best way to do so is to emit all state and counter updates as events — typically a complete set is emitted every time.
 You can then use `Actyx.observeLatest` to keep track of the latest known machine state.
 
-If for some reason you need to write partial updates (e.g. only a subset of counters) in some events, you can use a Fish to construct a full state from the partial updates.
-It is recommended to still emit the complete state from time to time. The Fish can treat that as a semantic snapshot:  
-This way, starting up the machine connector’s related Fishes will always be quick.
-
-The other performance consideration for machine connectors is that they produce data at a steady rate that is often higher than the data rates produced by human operators.
-Therefore you need to plan the available disk space according to update frequency times event size (compressed) for at least one year; we will release a future Actyx version with ephemeral event streams which will give you the possibility to place a fixed limit on disk usage and thus solve this issue permanently.
-
-### Production Data Acquisition
```

Comment:
Shouldn’t this headline stay in?

Reply:
I'm not sure how to formulate it without Fish/Pond. 
Context https://discord.com/channels/697349510409617408/1074986110024945705/1156571785735700580
Could you give me a quick overview on fish's activity under state snapshot?

Reply:
Per the conversation in Discord. This section will not be replaced with the `with AQL` approach because this section focuses on performance and makes claims about performance.